File: src/main/java/club/mindtech/mindbot/util/StrUtil.java
package club.mindtech.mindbot.util;

import java.util.Set;

public class StrUtil {

    public static String GAP = "⠀";

    public static String bold(String text) {
        return "**" + text.strip() + "**";
    }

    public static String italic(String text) {
        return "*" + text.strip() + "*";
    }

    public static String underline(String text) {
        return "__" + text.strip() + "__";
    }

    public static String code(String text) {
        return "`" + text.strip() + "`";
    }

    public static String stringify(Set<String> strings) {
        return strings.toString().replace("[", "").replace("]", "");
    }

    public static String zFill(int number, int length) {
        return String.format("%0" + length + "d", number);
    }
}

File: src/main/java/club/mindtech/mindbot/util/StrUtil.kt
package club.mindtech.mindbot.util

const val gap = "⠀"

fun bold(text: String): String = "**${text.trim()}**"

fun italic(text: String): String = "*${text.trim()}*"

fun underline(text: String): String = "__${text.trim()}__"

fun code(text: String): String = "`${text.trim()}`"

fun stringify(strings: Set<String>): String = strings.toString().replace("\\[]".toRegex(), "")

fun zFill(number: Int, length: Int): String = "$number".padStart(length, '0')
